src/resources/postcss/base/typography/_headings.pcss

```diff
@@ -0,0 +1,97 @@
+.tribe-events {
+	/* -------------------------------------------------------------------------
+	 * Heading: h1
+	 * ------------------------------------------------------------------------- */
+
+	.tribe-common-h1 {
+		color: var(--color-text-primary);
```

Comment:
Wonder if it would be good to set these up as mixins so they could be easily reused as needed, thoughts?

Reply:
Will look, as I don't remember exactly how I set it up. I do remember trying that, but since it was only used once I decided against it. I thought that if we needed the `h1` styles we would just apply the class. But it couldn't hurt to have it as a mixin.

Reply:
Totally up to you and you can probably feel it out as you go 👍